README.md

```diff
@@ -1,177 +1,30 @@
-# chart-verifier
+# **chart-verifier**: Rules based tool to certify Helm Charts
 
-`chart-verifier` is a tool that validates a Helm chart against a configurable list of checks; individual checks can be
-included or excluded through command line options. The default set of tests covers Red Hat’s recommendations.
+The **chart-verifier** CLI tool allows you to validate the Helm chart against a configurable list of checks. The tool ensures that the Helm charts include the associated metadata and formatting, and are distribution ready.
 
-Each check is independent and execution order is not guaranteed. Input is provided through options in
-the command line interface; currently the only input is the required `uri` option.
+The tool allows users to validate a Helm chart URL and provides a report where each check has a `positive` or `negative` result. A negative result from a check indicates a problem with the chart, which needs correction. It ensures that the Helm chart works seamlessly on Red Hat OpenShift and can be submitted as a certified Helm chart in the [Red Hat Helm Repository](https://github.com/openshift-helm-charts).
```

Comment:
should we use `OpenShift Helm Repository` instead of `Red Hat Helm Repository`?

Reply:
Changed to OpenShift